static/js/components/ClassificationList.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { useSelector, useDispatch } from "react-redux";
+import { Button } from "@material-ui/core";
+import Tooltip from "@material-ui/core/Tooltip";
+import GroupIcon from "@material-ui/icons/Group";
+
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+import * as sourceActions from "../ducks/source";
+import styles from "./ClassificationList.css";
+
+dayjs.extend(relativeTime);
+
+const ClassificationList = ({ isCandidate }) => {
+  const [hoverID, setHoverID] = useState(null);
+
+  const handleMouseHover = (id, userProfile, author) => {
+    if (userProfile.roles.includes("Super admin") || userProfile.username === author) {
+      setHoverID(id);
+    }
+  };
+
+  const handleMouseLeave = () => {
+    setHoverID(null);
+  };
+
+  const dispatch = useDispatch();
+  const source = useSelector((state) => state.source);
+  const candidate = useSelector((state) => state.candidate);
+  const obj = isCandidate ? candidate : source;
+  const userProfile = useSelector((state) => state.profile);
+  const acls = useSelector((state) => state.profile.acls);
+  let { classifications } = obj;
+
+  classifications = classifications || [];
+
+  const items = classifications.map(
+    ({ id, author_name, created_at, classification, probability, taxonomy_id, groups }) => (
+      <span
+        key={id}
+        className={styles.classification}
+        onMouseOver={() => handleMouseHover(id, userProfile, author_name)}
+        onMouseOut={() => handleMouseLeave()}
+        onFocus={() => handleMouseHover(id, userProfile, author_name)}
+        onBlur={() => handleMouseLeave()}
+      >
+        <div className={styles.classificationHeader}>
+          <span className={styles.classificationUser}>
+            <span className={styles.classificationUserName}>
+              {author_name}
+            </span>
+          </span>
+          &nbsp;
+          <span className={styles.classificationTime}>
+            {dayjs().to(dayjs(created_at))}
+          </span>
+          &nbsp;
+          <Tooltip title={groups.map((group) => group.name).join(", ")}>
+            <GroupIcon fontSize="small" style={{ paddingTop: "6px", paddingBottom: "0px" }} />
+          </Tooltip>
+        </div>
+        <div className={styles.wrap} name={`classificationDiv${id}`}>
+          <div className={styles.classificationMessage}>
+            {classification} (P={probability})
```

Comment:
Currently the classifications are displayed in two different parts of the page, header tags and the classification "comment list". I think only one of these is needed. I would opt for the header tags that aggregate all the classifications of a single object within a taxonomy into a single tooltip. The comment list has the downside of having different classifications not grouped by taxonomy, so trickier to track the classification history of an object within a single taxonomy. If the comment list is kept,

Reply:
It should also display the taxonomy and version that a classification was made under. But I would advocate for consolidating everything into the header tags

Reply:
I disagree. It's helpful to see all the classifications (as they change as more data arrives). We also need a frontend way for people to delete classifications (like comments). I think it's also helpful to have a "quicklook" view at the top.

Reply:
> It should also display the taxonomy and version that a classification was made under. But I would advocate for consolidating everything into the header tags

Now showing classification author and the name of the taxonomy